docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-slurm/install-on-slurm.rst

```diff
@@ -155,7 +156,15 @@ fulfilled and configured, install and configure the Determined master:
    see :ref:`slurm-image-config`.
 
 #. Log into Determined, see :ref:`users`. The Determined user must be linked to a user on the HPC
-   cluster, see :ref:`run-as-user`.
+   cluster. If logged with a Determined admininstrator account, the following example creates a
+   Determined user account properly linked for the current user. For more details see
```

Comment:
The image in the summary doesn't match what's in the diffs shown here.

![image](https://user-images.githubusercontent.com/90728398/206226258-91418457-8c71-42c6-8ce6-d2ffec5d69a2.png)

Maybe it's just a caching issue on my browser?

Reply:
Hmm.  I pushed something, but I see that change wasn't in.   Just pushed it again.